expressEngine/user/service/user.js

```diff
@@ -7,6 +7,9 @@ import { User } from '../../model/index';
 // import messages
 import { error } from '../../cms/user/index';
 
+// import utils
+import { UserHelper } from '../utils';
```

Comment:
I would suggest keeping UserHelper in camelCase.

Reply:
Sir UserHelper is class